### ISSUE TITLE
[enhancement] Add some colors for easier visibility

### ISSUE THREAD

**Issue by rieje** [ilpianista/arch-audit#29]
Add some colors for easier visibility. And perhaps a link to where more information on the vulnerability can be found (can be just the generic page) or a hyperlink for each specific vulnerability. These can of course be options.

**Comment by clarcharr** [comment]
Dupe of #25.

**Comment by ilpianista** [comment]
Yes, it's partially a duplicate of #25. I renamed it so it doesn't conflict.
